### PR TITLE
Wizard

### PR DESCRIPTION
**Pull request by SteffenHeu**
- adjusted Oribtrap parameters
- added licenses
- force disable of isotope mass detection in auto mass detector

### CODE REVIEW

File: src/main/java/io/github/mzmine/modules/tools/batchwizard/defaults/DefaultMsParameters.java [hunk 1]
```diff
@@ -13,12 +31,12 @@
       1E2, 1E3, new MZTolerance(0.005, 10), new MZTolerance(0.005, 3), new MZTolerance(0.005, 8));
 
   public static final DefaultMsParameters defaultOrbitrapPositiveParameters = new DefaultMsParameters(
-      1E4, 5E3, 5E4, new MZTolerance(0.005, 5), new MZTolerance(0.005, 3),
-      new MZTolerance(0.005, 5));
+      1E5, 3E3, 3E5, new MZTolerance(0.0015, 10), new MZTolerance(0.0015, 5),
```
Comment (tomas-pluskal) [inline src/main/java/io/github/mzmine/modules/tools/batchwizard/defaults/DefaultMsParameters.java:34]: Robin set the 1E5 noise level as default for the workshop, to make processing faster. But I think 1E4 is better for real applications 1E4 is better to catch small peaks and isotopes.

File: src/main/java/io/github/mzmine/modules/dataprocessing/featdet_massdetection/auto/AutoMassDetector.java [hunk 2]
```diff
@@ -44,16 +62,21 @@
   }
 
   private ParameterSet getExactParam(ParameterSet autoParam) {
-    exactMassDetectorParameters = new ExactMassDetectorParameters();
+    exactMassDetectorParameters = (ExactMassDetectorParameters) (new ExactMassDetectorParameters())
+        .cloneParameterSet();
     exactMassDetectorParameters.getParameter(ExactMassDetectorParameters.noiseLevel)
         .setValue(autoParam.getParameter(AutoMassDetectorParameters.noiseLevel).getValue());
+    exactMassDetectorParameters.setParameter(ExactMassDetectorParameters.detectIsotopes, false);
```
Comment (tomas-pluskal) [inline src/main/java/io/github/mzmine/modules/dataprocessing/featdet_massdetection/auto/AutoMassDetector.java:69]: The detectIsotopes is actually a nice feature for the high-res Orbitrap data, might be worth setting it to true (with default C, H, N, O, S, P elements)

**Comment by SteffenHeu** [comment]
Also changed the default paint scale and made all color palettes available on first load

**Comment by SteffenHeu** [comment]
Ok. I will change it.

What about the tolerances, should they also stay narrower as they were before, or should htey be widened?

**Comment by tomas-pluskal** [comment]
The tolerance looks good to me

**Comment by SteffenHeu** [comment]
Ok. I will remove P though, as it is monoisotopic.

**Pull request merged**